### PR TITLE
change game options

### PR DESCRIPTION
**Pull request by kudukacar**
Give user the game choices of human vs. human, human vs. computer, and computer vs. computer.

### CODE REVIEW

File: lib/game.rb [hunk 1]
```diff
@@ -50,11 +50,16 @@ def selection(game_options_descriptions)
   position_validator = PositionValidator.new(board)
   parse_input = ParseInput.new
   user = User.new(display: display, parse_input: parse_input)
-  human_player = HumanPlayer.new(user: user, token: "X", validator: position_validator)
-  computer_player = ComputerPlayer.new(token: "O", board: board)
+  X = "X"
+  O = "O"
```
Comment (damonkelley) [inline lib/game.rb]: Where does this constant get defined? Is it defined globally? If this was inside a class, it would be localized.

```ruby
class SomeClass
  X = "X"
end

SomeClass::X
```

I wonder if it needs to be constant here in the `main`. Is there a benefit over using a normal variable?

Comment (kudukacar) [reply lib/game.rb]: I think normal variable is fine.  No constant needed.

File: spec/human_player_spec.rb [hunk 2]
```diff
@@ -5,20 +5,20 @@
...
   describe "#selection" do
     it "returns the player's valid selection" do
       token = ""
       validator = ValidatorWithNoMethods.new
-      user = UserWithOneValidInput.new
+      user = UserWithFourValidInput.new
       player = HumanPlayer.new(token: token, user: user, validator: validator)
 
-      expect(player.selection).to eq(1)
+      expect(player.selection).to eq(4)
```
Comment (hyoyou) [inline spec/human_player_spec.rb]: hmm.. curious, why did you need to change this to a 4?

Comment (kudukacar) [reply spec/human_player_spec.rb]: I thought I could get one by you.  For no other reason than to give it a unique name.  I should just pull it out into another file like I did FakeDisplay.
